src/assisted_test_infra/test_infra/tools/load_system.py

```diff
@@ -0,0 +1,124 @@
+import weakref
+from abc import ABC, abstractmethod
+from string import Template
+
+from service_client import log
+
+
+class Command(ABC):
+    base_command = None
+    redirect = " > /dev/null &"
+
+    def __init__(self, nodes: list):
+        """Initialize weakref because dictionay keys are node object
+        :param nodes: node is the command invoker run_command
+        """
+        self.node_undu_commands = weakref.WeakKeyDictionary()
+        self.nodes = nodes
+
+    @abstractmethod
+    def execute(self):
+        pass
+
+    def undu(self):
+        for node in self.nodes:
+            cmd = self.node_undu_commands[node]
+            log.info(f"{node.name}  background execute: {cmd}")
+            node.run_command(cmd, background=True)
+
+
+class LoadCPU(Command):
```

Comment:
Maybe a way to simulate cpu overload is to limit cpu usage by limiting cpu instead of creating load.  Cgroups can provide it,or `cpulimit`

Reply:
cpulimit not installed by default in our  CoreOS
cgroups never tried...
Not sure . 
cat /dev/urandom | gzip -9 | gzip -d | gzip -9 | gzip -d > /dev/null
This command will create the process on every cpu core .  we install out automation with minimal cores allowed by preflight .